packages/yoroi-extension/app/App.js

```diff
@@ -126,38 +128,79 @@ class App extends Component<Props, State> {
 
     return (
       <div style={{ height: '100%' }}>
-        <LayoutProvider layout={currentTheme}>
-          <ThemeProvider theme={muiTheme}>
-            <CssBaseline />
-            {globalStyles(muiTheme)}
-            <ThemeManager cssVariables={themeVars} />
-            {/* Automatically pass a theme prop to all components in this subtree. */}
-            <IntlProvider {...{ locale, key: locale, messages: mergedMessages }}>
-              {this.getContent()}
-            </IntlProvider>
-          </ThemeProvider>
-        </LayoutProvider>
+        <QueryClientProvider client={queryClient}>
+          <MetricsProvider storage={metricsStorage} metrics={amplitudeClient}>
+            <LayoutProvider layout={currentTheme}>
+              <ThemeProvider theme={muiTheme}>
+                <CssBaseline />
+                {globalStyles(muiTheme)}
+                {/* Automatically pass a theme prop to all components in this subtree. */}
+                <ThemeManager cssVariables={themeVars} />
+                <IntlProvider {...{ locale, key: locale, messages: mergedMessages }}>
+                  {this.getContent()}
+                </IntlProvider>
+              </ThemeProvider>
+            </LayoutProvider>
+          </MetricsProvider>
+        </QueryClientProvider>
       </div>
     );
   }
 
-  getContent: void => ?Node = () => {
-    const { stores, actions, history } = this.props;
-    if (this.state.crashed === true) {
-      return <CrashPage />;
-    }
-    if (stores.serverConnectionStore.isMaintenance) {
-      return <MaintenancePage stores={stores} actions={actions} />;
-    }
+  getContent: void => Node = () => {
```

Comment:
Is this `this.getContent` really necessary, any reason to don't render the component directly?